zerver/tests/test_events.py

```diff
@@ -1525,7 +1525,7 @@ def test_default_streams_events_guest(self) -> None:
     def test_muted_topics_events(self) -> None:
         muted_topics_checker = self.check_events_dict([
             ('type', equals('muted_topics')),
-            ('muted_topics', check_list(check_list(check_string, 2))),
+            ('muted_topics', check_list(check_list(None))),
```

Comment:
`None`, because the function can only validate if all items are of the same type.
There is a todo for `realm_filters` in the same file: `TODO: validate tuples in the list`

Reply:
Just to make sure I understand this, we're changing a 2-element list to a 3-element list?  I think we should at least preserve the `, 2` -> `, 3` part of the check here.  

In any case, this is a wire format API change that we need to run by the mobile folks.  @gnprice how does the mobile app parse the `muted_topics` event type?  Would changing the format of that to add an extra element to the list be a problem?

Reply:
Connected zulip-mobile with dev server, and everything seems normal, the topics gets displayed as it should and I am able to mute topics. (Attached the testing below)

![zulip_mobile_debug](https://user-images.githubusercontent.com/40122794/74605958-7bf23500-50f2-11ea-8c57-8249b6415b89.gif)